tests/UnitTests/ImageProcessing/DIPCorr2DTest.cpp

```diff
@@ -0,0 +1,89 @@
+#include "../../../include/ImageProcessing/Kernels.h"
```

Comment:
You can use the `#include "ImageProcessing/Kernels.h"` directly, the CMake file helps you find the include directory.